_posts/en/newsletters/2023-02-08-newsletter.md

```diff
@@ -54,12 +54,80 @@ Bitcoin infrastructure software.
 meeting, highlighting some of the important questions and answers.  Click on a
 question below to see a summary of the answer from the meeting.*
 
-FIXME is a PR by FIXME that FIXME:LarryRuane
+[Track AddrMan totals by network and table, improve precision of adding fixed seeds][review club 26847]
+is a PR by Martin Zumsande, co-authored by Amiti Uttarwar, that
+allows the Bitcoin Core client to more reliably find outbound peers
```

Comment:
maybe a bit weaker, "... in certain situations" or so.
The fix in this PR really only improves the `-onlynet` situation described in Q2, not finding outbound peers in general.